src/main/java/lotto/service/LottoService.java

```diff
@@ -0,0 +1,10 @@
+package lotto.service;
+
+import lotto.model.Lotto;
+
+import java.util.List;
+
+public interface LottoService {
+    List<Lotto> purchaseLotto(int amount);
```

Comment:
List를 반환하는데, 메소드는 단수형입니다. 복수형으로 바꾸는게 적절해 보입니다
```suggestion
    List<Lotto> purchaseLottos(int amount);
```